src/lib/network/clients/save_network.rb

```diff
@@ -220,6 +204,33 @@ def copy_from_instsys
       nil
     end
 
+    WICKED_DHCP_PATH = "/var/lib/wicked/".freeze
+    WICKED_DHCP_FILES = ["duid.xml", "iaid.xml", "lease*.xml"].freeze
+    DHCP_CLIENT_PATH = "/var/lib/dhcpcd/".freeze
+    DHCP_CLIENT_CACHE = "dhcpcd-*.cache".freeze
+    DHCPv6_CLIENT_CACHE_PATH = "/var/lib/dhcpv6".freeze
```

Comment:
BTW, missing "/" at the end?

Reply:
In this case it just a file not a dir

Reply:
Just checked my system, there is no /var/lib/dhcpv6 but /var/lib/dhcp6

Reply:
Ups, you are right the original code is a directory but it is working at all? Isn't it doing a:

```
/bin/cp -p /var/lib/dhcpv6/ /mnt/var/lib/dhcpv6
```

without recursie option looks strange

Reply:
Also the bug reference of copying the dhclient files seems to be wrong

Reply:
And this is the man of the client used in SLE11 so probably now /var/lib/dhcp6 should be used instead as you commented

https://www.unix.com/man-page/suse/8/dhcp6c/